Specs/Number/Hindi/NumberModel.json

```diff
@@ -0,0 +1,2463 @@
+[
+  {
+    "Input": "192.",
```

Comment:
New tests cases for scenarios using the Indian numbering system (e.g., lakh, crore) will come in future PRs, correct?

Reply:
yes, we will add those in the respective subtype PRs